src/index.ts

```diff
@@ -199,8 +208,22 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
       throw error;
     }
   }
-  EppoJSClient.initialized = true;
-  return EppoJSClient.instance;
+}
+
+export async function prefetchConfig(config: IClientConfig) {
+  await initClient(config);
+
+  await EppoJSClient.instance.fetchFlagConfigurations();
```

Comment:
oh nice! I see now that we can do this because it's public ✔️

Reply:
Note this won't fetch if the underlying store is expired ([source](https://github.com/Eppo-exp/js-client-sdk-common/blob/main/src/client/eppo-client.ts#L162))

This works well with what you have going on; just make sure you're aware as it's not intuitive.